src/tag/selectable.jsx

```diff
@@ -51,6 +51,16 @@ class Selectable extends Component {
         bindCtx(this, ['handleClick']);
     }
 
+    static getDerivedStateFromProps(props, state) {
```

Comment:
we need to import `react-lifecycles-compat` to make sure it can work on React v16.0-16.2
please run :
`npm install --save react-lifecycles-compat`

```
// add
import { polyfill } from 'react-lifecycles-compat';

class Selectable extends Component {
   ...
}
// add
polyfill(Selectable);

export default Selectable;
```